# hsv param: hand
H_MIN_HAND: 0
H_MAX_HAND: 179
S_MIN_HAND: 0
S_MAX_HAND: 255
V_MIN_HAND: 0
V_MAX_HAND: 70

# hsv param: obj
H_MIN_OBJ: 0
H_MAX_OBJ: 8
S_MIN_OBJ: 0
S_MAX_OBJ: 255
V_MIN_OBJ: 0
V_MAX_OBJ: 255